src/algebra/lie/ideal_operations.lean

```diff
@@ -171,6 +171,25 @@ begin
   exact lie_submodule.lie_mem_lie _ _ fy₁ fy₂,
 end
 
+lemma map_bracket_eq {I₁ I₂ : lie_ideal R L} (h : function.surjective f) :
+  map f ⁅I₁, I₂⁆ = ⁅map f I₁, map f I₂⁆ :=
```

Comment:
Isn't one of the two inequalities always true?

Reply:
Yes indeed, cast your eyes up to line 163!

Reply:
Ooh nice, but it doesn't help golfing here? That's (a bit) surprising.

Reply:
It probably does :-) Let me see if I can do that!

Reply:
Now done 🎊 Thanks for the suggestion.